snap/snapcraft.yaml

```diff
@@ -156,13 +156,30 @@ parts:
       - libuv1
     build-packages:
       - libuv1-dev
+      - pkg-config
```

Comment:
Not sure why this works when building the full snap, but when only building the parts relevant to the K8s binaries, this package isn't installed.